src/native/windows/apps/prunsrv/prunsrv.c

```diff
@@ -42,6 +42,7 @@
 #define STDOUT_FILENO 1
 #define STDERR_FILENO 2
 #define ONE_MINUTE    (60 * 1000)
+#define ONE_MINUTE_SEC 60
```

Comment:
`ONE_MINUTE_SEC` looks weird every time I look at it even when I know what it means. Maybe
```
ONE_MINUTE_AS_MILLIS
ONE_MINUTE_AS_SEC
```